src/locales/en/en.json

```diff
@@ -3574,5 +3574,20 @@
 	},
 	"nl_query_example_3": {
 		"text": "Find me some dinner spots around Vancouver?"
+	},
+	"nl_loader_sample_text_1": {
```

Comment:
This is cool, but it's not the best fit for geocoding operation. We can push these changes now, and decide on exact messaging later. We probably want something more concrete and mock response from LLM while waiting for an actual response.